pkg/interface/src/views/components/Tab.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Box, Text, Row } from "@tlon/indigo-react";
+
+export const Tab = ({ selected, id, label, setSelected }) => (
+  <Box
+    py={2}
+    borderBottom={1}
+    borderBottomColor={selected === id ? "black" : "washedGray"}
+    px={2}
+    flexGrow={1}
+    display="flex"
+    alignItems="center"
+    justifyContent="center"
+    onClick={() => setSelected(id)}
+  >
+    <Text color={selected === id ? "black" : "gray"}>{label}</Text>
+  </Box>
+);
+
+export const Tabs = ({ children, ...rest }: Parameters<typeof Row>[0]) => (
+  <Row
+    bg="white"
+    mb={2}
+    justifyContent="stretch"
+    alignItems="flex-end"
+    {...rest}
+  >
```

Comment:
Needs `cursor: pointer`.

Reply:
Also the notebook list items aren't respecting the margin bottom here. Is it an inline item?
<img width="300" alt="Screen Shot 2020-09-22 at 1 33 30 PM" src="https://user-images.githubusercontent.com/20846414/93917039-6ec8a480-fcd8-11ea-9faf-7a85d5215a0a.png">